commands/pm/pm.drush.inc

```diff
@@ -1059,9 +1059,11 @@ function drush_pm_enable_validate() {
       // Check to see if the module can be installed/enabled (hook_requirements).
       // See @system_modules_submit
       if (!drupal_check_module($module)) {
-        unset($modules[$key]);
         drush_set_error('DRUSH_PM_ENABLE_MODULE_UNMEET_REQUIREMENTS', dt('Module !module doesn\'t meet the requirements to be enabled.', array('!module' => $module)));
         _drush_log_drupal_messages();
+        if (count($modules) > 1) {
+          drush_set_error('DRUSH_PM_ENABLE_MODULE_UNMEET_REQUIREMENTS', dt('None of these !modules is enabled', array('!modules' => join(', ', array_keys($modules)))));
+        }
         return FALSE;
```

Comment:
Should we return or continu enabling the other modules. This line is from

> 68684e83 (Greg Anderson       2011-01-05 06:28:44 +0000 1067)         return FALSE;

Can we continue enabling the others?